src/assets/css/hide.css

```diff
@@ -0,0 +1,28 @@
+.loader{
```

Comment:
Please use an <ion-spinner /> component from Ionic Framework

Reply:
The Ionic Framework provides a component for this already that matches the light and dark themes of the project https://ionicframework.com/docs/api/spinner